app/views/places/_form.html.erb

```diff
@@ -12,18 +12,18 @@
       </div>
     <% end %>
 
-    <div class="field">
+    <div class="form-group">
       <%= form.label Place.human_attribute_name("name") %>
-      <%= form.text_field :name %>
+      <input type="name" class="form-control" %>
     </div>
 
-    <div class="field">
+    <div class="form-group">
       <%= form.label Place.human_attribute_name("adres") %>
-      <%= form.text_area :address %>
+      <input type="address" class="form-control" %>
     </div>
 
-    <div class="actions">
-      <%= form.submit %>
+    <div>
+      <button type="submit" class="btn btn-primary mb-2"><%= t(".submit")%></button>
```

Comment:
And here.

Reply:
Is it a good way to do this: 
<%= button_to [:make_happy, @user] do %>
  Make happy <strong><%= @user.name %></strong>
<% end %>
# => "<form method="post" action="/users/1/make_happy" class="button_to">
#      <button type="submit">
#        Make happy <strong><%= @user.name %></strong>
#      </button>
#    </form>"

??

Reply:
But why you need the button there? I think it is enough to have input type submit. What you would like to get with this button?